thirdparty/runtime_environments.go

```diff
@@ -58,3 +64,38 @@ func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string
 	}
 	return filteredImages, nil
 }
+
+// getOSInfo returns a list of operating system changes (name and version) from the corresponding AMI id.
+func (c *RuntimeEnvironmentsClient) getOSInfo(ctx context.Context, amiId string, page string, limit string) ([]OSInfo, error) {
```

Comment:
nit. Use `amiID` according to the [Go Coding Conventions](https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit#heading=h.3rcuk9z7pv53) (Naming section)
```suggestion
func (c *RuntimeEnvironmentsClient) getOSInfo(ctx context.Context, amiID string, page string, limit string) ([]OSInfo, error) {
```